src/Hooks/ResizeWindow.js

```diff
@@ -0,0 +1,17 @@
+import { useState, useEffect } from "react";
+
+export function useResizeWindow() {
+  const [currentWidth, setCurrentWidth] = useState();
```

Comment:
One more thing, could be more consistent to return a `boolean`, that checks if `currentWidth > 992`.
This way you can just check the boolean in the other components that use the hook.